verify/update-bazel.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# TODO(spxtr): Pin to gazel version once mikedanese/gazel#14 is in.
```

Comment:
I need gazel v5 to happen on gopkg.in before I can execute. Updated the comments.

We can vendor if we'd like, but it doesn't seem too important for now.